### PR TITLE
Redesign footer

### PR DESCRIPTION
**Pull request by SvitlanaYaroslavicka1**
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/ita-social-projects/EventsExpress-Frontend/projects/1)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/components/footer/footer.js [hunk 1]
```diff
@@ -1,13 +1,15 @@
 import React from "react";
-import "./footer.css";
 import { Link } from "react-router-dom";
+import Logo from "../shared/Logo/logo";
+import "./footer.scss";
 
 const Footer = () => {
   const links = [
```
Comment (VadymKharyton) [inline src/components/footer/footer.js]: Move this variable out of component

File: src/components/footer/footer.js [hunk 2]
```diff
@@ -18,20 +20,40 @@ const Footer = () => {
 
   return (
     <footer className="custom-footer">
+      <div className="custom-footer__logo">
```
Comment (VadymKharyton) [inline src/components/footer/footer.js:26]: Redundant div. If you want to customize Logo with styles, pass className prop directly to Logo

File: src/components/footer/footer.js (hunk 1 above)
Comment (SvitlanaYaroslavicka1) [reply src/components/footer/footer.js]: Done

File: src/components/footer/footer.js (hunk 2 above)
Comment (SvitlanaYaroslavicka1) [reply src/components/footer/footer.js:26]: I need this div for logo position.
But I delete className for Logo component

Comment (SvitlanaYaroslavicka1) [reply src/components/footer/footer.js:23]: But I delete className for component Logo

**Review by VadymKharyton — APPROVED** [review]

**Pull request merged**